GPU/GLES/GLES_GPU.cpp

```diff
@@ -565,6 +565,8 @@ void GLES_GPU::CheckGPUFeatures() {
 	if (gl_extensions.GLES3 || !gl_extensions.IsGLES)
 		features |= GPU_SUPPORTS_TEXTURE_LOD_CONTROL;
 
+	features |= GPU_SUPPORTS_ANISOTROPY;
```

Comment:
I see we can check GL_EXT_texture_filter_anisotropic, seems fine for mobile and desktop, right?  I see a Hisilicon Technologies Immersion.16 reporting GL_OES_anisotropic, but I suppose that's not a real thing.

-[Unknown]